### ISSUE TITLE
Bug: Volume up 100% bug

### ISSUE THREAD

**Issue by sandrzejewskipl** [eritislami/evobot#944]
**Describe the bug**
When DEFAULT_VOLUME is set in docker-compose, volume up emoji sets volume to 100%, not x+10%.

**How To Reproduce**
1. Set DEFAULT_VOLUME in docker compose (e.g DEFAULT_VOLUME: 30)
2. Use Volume UP emoji
 
**Expected behavior**
Volume up +10%

**Environment (add if possible)**
Docker + docker compose

**Additional information & screenshots**
![image](https://user-images.githubusercontent.com/35699058/134772184-bbbde61f-251e-48fa-9d7a-3baf3382ae66.png)

This issue doesn't affect volume down emoji (it works normal).

**Comment by drewburr** [comment]
This is happening to myself as well. Using version 0.15.2, running in Docker w/ Kubernetes. 

Default volume is set to 20. When the bot joins, volume is properly set to 20. The first time a user increases the bot volume using reactions, it jumps to 100. This does not happen if the volume is first decreased, but does still happen if the volume is set using the `/volume` command, regardless of what is set to.

```
# Bot connected for first time

/volume
@drewburr, 🔊 The current volume is: 20%

@drewburr 🔊 increased the volume, the volume is now 100%

# Disconnected and rejoined

/volume
@drewburr, 🔊 The current volume is: 20%

@drewburr 🔉 decreased the volume, the volume is now 10%

@drewburr 🔊 increased the volume, the volume is now 20%

# Disconnected and rejoined

/volume
@drewburr, 🔊 The current volume is: 20%

/volume 20
Volume set to: 20%

@drewburr 🔊 increased the volume, the volume is now 100%

# Disconnected and rejoined

/volume
@drewburr, 🔊 The current volume is: 20%

/volume 30
Volume set to: 30%

@drewburr 🔊 increased the volume, the volume is now 100%
```

**Comment by drewburr** [comment]
Looking into this more, I can confirm that this is specific to the Docker implementation. To be more specific, this is the result of environment variables being used rather than 'config.json'. This is reproducible outside of the Docker container by using:

```shell
export TOKEN=<discord_token>
export YOUTUBE_API_KEY=<key>
export DEFAULT_VOLUME=20
node index.js
```

**Comment by drewburr** [comment]
Found our bug. Environment variable are being retrieved as strings, which means `'20' + 10 == '2010'`. Fixing this is going to have to be a judgement call. In my opinion, we should be converting values when they are sourced from the env, inside of `util/Util.js`. This would also need to happen for other non-string values, such as PRUNING and STAY_TIME. This might also be a fix for #950.

Test code:

```javascript
case "🔊":
          queue.textChannel.send("Pre-processing: ".concat(queue.volume)); // added logging
          reaction.users.remove(user).catch(console.error);
          if (queue.volume == 100) return;
          if (!canModifyQueue(member)) return i18n.__("common.errorNotChannel");
          queue.textChannel.send("Post-pre-processing: ".concat(queue.volume)); // added logging
          queue.textChannel.send("Addition: ".concat(queue.volume + 10)); // added logging
          queue.textChannel.send("Min: ".concat(Math.min(queue.volume + 10, 100))); // added logging
          queue.volume = Math.min(queue.volume + 10, 100);
          queue.textChannel.send("Post-processing: ".concat(queue.volume)); // added logging
```

Logs from Discord text chat:

```
!v
@drewburr, 🔊 The current volume is: 20%

Pre-processing: 20
Post-pre-processing: 20
Addition: 2010
Min: 100
Post-processing: 100
@drewburr 🔊 increased the volume, the volume is now 100%
```

POC on the CLI:

```shell
node
> var value = '20';
undefined
> value
'20'
> value + 10
'2010'
> value = 20;
20
> value + 10
30
> 
```
